Python/bytecodes.c

```diff
@@ -2571,6 +2571,16 @@ dummy_func(
             JUMP_BACKWARD_NO_INTERRUPT,
         };
 
+        pseudo(JUMP_IF_FALSE, (cond -- cond)) = {
+            JUMP_FORWARD,
+            JUMP_BACKWARD,
```

Comment:
I'm cheating here, it's not the right definition. I need to think how to do this properly in the generator. I think we need a version of pseudo that takes a sequence of instructions rather than a set of options. I'll think about that tomorrow.